docs/src/modules/components/LocalizationTable.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { styled } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableHead from '@mui/material/TableHead';
+import TableCell from '@mui/material/TableCell';
+import TableRow from '@mui/material/TableRow';
+import Typography from '@mui/material/Typography';
+import Link from 'docs/src/modules/components/Link';
+import MarkdownElement from 'docs/src/modules/components/MarkdownElement';
+
+const Root = styled('div')(({ theme }) => ({
+  position: 'relative',
+  overflow: 'hidden',
+  width: '100%',
+  height: 26,
+  borderRadius: 2,
+  '&.low': {
+    color: (theme.vars || theme).palette.error.contrastText,
+    backgroundColor: (theme.vars || theme).palette.error.light,
+    '& .progress-bar': {
+      backgroundColor: (theme.vars || theme).palette.error.dark,
+    },
+  },
+  '&.medium': {
+    color: (theme.vars || theme).palette.warning.contrastText,
+    backgroundColor: (theme.vars || theme).palette.warning.light,
+    '& .progress-bar': {
+      backgroundColor: (theme.vars || theme).palette.warning.dark,
+    },
+  },
+  '&.high': {
+    color: (theme.vars || theme).palette.success.contrastText,
+    backgroundColor: (theme.vars || theme).palette.success.light,
+    '& .progress-bar': {
+      backgroundColor: (theme.vars || theme).palette.success.dark,
+    },
+  },
+}));
+
+const Value = styled('div')({
+  position: 'absolute',
+  lineHeight: '24px',
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'center',
```

Comment:
One last minor nitpick: I think this element could use the following addition. 🤔 

```suggestion
  justifyContent: 'center',
  alignItems: 'center',
```